addons/mail/models/mail_poll.py

```diff
@@ -47,7 +47,7 @@ def _to_store_defaults(self, target):
             Store.Many("option_ids"),
             "poll_end_dt",
             "poll_question",
-            "start_message_id",
+            Store.One("start_message_id") if with_start_message_id else "start_message_id",
```

Comment:
I would not send the field at all. `start_message_id` is the inverse of `started_poll_ids` so the information is already present in that case.